src/libs/auth.ts

```diff
@@ -46,23 +47,30 @@ export const getOidcConfig = () => {
     stateStore: new WebStorageStateStore({ store: getLocalStorage() }),
     userStore: new WebStorageStateStore({ store: getLocalStorage() }),
     automaticSilentRenew: true,
-    // Leo's setCookie interval is currently 5 min, set refresh auth then 5 min 30 seconds to gurantee that setCookie's token won't expire between 2 setCookie api calls
+    // Leo's setCookie interval is currently 5 min, set refresh auth then 5 min 30 seconds to guarantee that setCookie's token won't expire between 2 setCookie api calls
     accessTokenExpiringNotificationTimeInSeconds: 330,
     includeIdTokenInSilentRenew: true,
     extraQueryParams: { access_type: 'offline' },
+    redirect_uri: '', // this field is not being used currently, but is expected from UserManager
   };
 };
 
 const isGoogleAuthority = () => {
   return _.startsWith('https://accounts.google.com', authStore.get().oidcConfig.authorityEndpoint);
 };
 
-const getAuthInstance = () => {
+const getAuthInstance = (): any => {
```

Comment:
```suggestion
const getAuthInstance = () => {
```

Just to avoid adding to the code base any more `any`s than necessary, let's remove this. Its inferred return type is `any`, so it won't make a difference for type checking.

Reply:
I sort of like having the explicit any to remind myself that we should more towards specific types if possible, but I understand that the omission of the any is equivalent. I made the two changes to remove them in the meantime.

Reply:
My concern with this `any` is that when we add a type for authStore's state...
- without this `any`, this function's return type will be inferred from authStore's type
- with this `any`, this function's return type will be `any` and we'll have to remember to update this function as well in order to realize the type safety gains of adding a type to authStore

Reply:
That's a good point, I agree with that 👍